src/values/SymbolValue.js

```diff
@@ -9,11 +9,11 @@
 
 /* @flow */
 
-import { PrimitiveValue } from "./index.js";
+import { PrimitiveValue, AbstractValue } from "./index.js";
 import type { Realm } from "../realm.js";
 
 export default class SymbolValue extends PrimitiveValue {
-  constructor(realm: Realm, desc?: string, intrinsicName?: string) {
+  constructor(realm: Realm, desc?: string | AbstractValue, intrinsicName?: string) {
```

Comment:
So do you think that the description can never be undefined? In src/intrinsics/ecma262/Symbol.js lines 29/30 it appears that we are allowing for the case where the description is undefined.